DetoxSync/DetoxSync/Utils/_DTXTimerTrampoline.m

```diff
@@ -112,47 +112,24 @@ - (void)setDisplayLink:(CADisplayLink*)displayLink
 #endif
 }
 
-- (void)fire:(id)timer
+- (void)fire
```

Comment:
The following changes are a bit more involved.

Firstly, there's no need for the `timer` parameter - we already have a reference to that as an ivar.

Reply:
I can understand why the `timer` argument is redundant.